### PR TITLE
improve playwright tests with refactoring and clean up

### PR DESCRIPTION
**Pull request by prakashsvmx**
improve playwright tests with refactoring and clean up

Currently  running the test locally multiple times requires manual clean up ( removal of bucket once test verification step is done)

This PR adds clean up functionality and also the steps based tests.

to run/test locally,
```
# Terminal 1 Start MinIO
CI=true  MINIO_ROOT_USER=minioadmin MINIO_ROOT_PASSWORD=minioadmin minio server /tmp/site1{1...4}  --address ":9000" --console-address ":9001"


# Terminal ( start console server)
make install 
CONSOLE_MINIO_SERVER=http://localhost:9000 ./console server

# Terminal 3 ( run playwright tests)
cd portal-ui
npx playwright test

# To debug a specific test file via visual runner
npx playwright test lifecycle.spec.ts --ui

```

### CODE REVIEW

File: portal-ui/e2e/lifecycle.spec.ts [hunk 1]
```diff
@@ -17,37 +17,113 @@
...
+    page,
+  }) => {
+    const bucketLocatorEl = `${bucketListLocPrefix}-${nonVersionedBucketName}`;
+
+    await test.step(`Create bucket`, async () => {
+      await page.locator("#create-bucket").click();
+      await page.getByLabel("Bucket Name*").click();
+      await page.getByLabel("Bucket Name*").type(nonVersionedBucketName);
+      await page.getByRole("button", { name: "Create Bucket" }).click();
+    });
```
Comment (cesnietor) [inline portal-ui/e2e/lifecycle.spec.ts:92]: since these blocks are being reused would it be possible to abstract them so that we can keep them DRY?

Comment (prakashsvmx) [reply portal-ui/e2e/lifecycle.spec.ts]: I have updated. thank you for the feedback @cesnietor :+1:

**Review by cesnietor — APPROVED** [review]
Thanks for addressing the feedback, LGTM!

**Review by bexsoft — APPROVED** [review]
LGTM

**Pull request merged**